### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by dittodhole** [chriseldredge/Lucene.Net.Linq#20]
```
var articlesByJohn = from a in articles
                      where a.Author == "John Doe" && a.PublishDate > threshold
                      orderby a.Title
                      select a;
```

This expression gives me in the debugger a "Children could not be evaluated"-error, and when going for a `.ToList()` (or `.Count()`, ...), the returned collection is empty.

**Comment by dittodhole** [comment]
Interesting:

```
var firstArticle = articles.First();
var firstAuthor = firstArticle.Author;
// firstAuthor == "John Doe" : true

var articlesByJohn = from article in articles
                     where article.Author == "John Doe"
                     where article.PublishDate > threshold
                     orderby article.Title
                     select article; // empty

var articlesByJohn = from article in articles
                     where string.Equals(article.Author, "John Doe")
                     where article.PublishDate > threshold
                     orderby article.Title
                     select article; // not empty
```

**Comment by chriseldredge** [comment]
Thanks for reporting. After some changes to the library, it is necessary to specify an analyzer for each field or else a default KeywordAnalyzer will be used. Fixed in cdbc4bccb83701a828a8dd2bdaa7f3ade8c32a5a.
